share/spice/dogo_news/dogo_news.js

```diff
@@ -0,0 +1,50 @@
+(function(env) {
+    "use strict";
+
+    env.ddg_spice_dogo_news = function(api_result) {
+        if (!api_result || !api_result.results || !api_result.results.length) {
+            return Spice.failed('dogo_news');
+        }
+
+        // Get original query.
+        var script = $('[src*="/js/spice/dogo_news/"]')[0],
+            source = $(script).attr("src"),
+            query = decodeURIComponent(source.match(/dogo_news\/([^\/]+)/)[1]);
+    
+        Spice.add({
```

Comment:
I just remembered we have a built-in News template that we use for our own News Instant Answer. If we update the code here as follows it will use the correct template. You'll notice the images are gone. For now we'll leave them out but we're working to bring image support to the new template and once that happens we'll be able to use your images as well:

```js
DDG.require('moment.js', function(){
            Spice.add({
                id: 'dogo_news',
                name: 'Kids News',
                data: api_result.results,
                meta: {
                    sourceName: 'DOGOnews',
                    sourceUrl: 'http://www.dogonews.com/search?query=' + encodeURIComponent(query) + '&ref=ddg',
                    itemType: 'kids news articles'
                },
                normalize: function(item) {
                    var thumb = item.hi_res_thumb || item.thumb;
                    return {
                        title: item.name,
                        source: item.author,
                        url: item.url,
                        excerpt: item.summary,
                        relative_time: moment(item.published_at).fromNow()
                    };
                },
                templates:{
                    item: 'news_item'
                }
            });
        });
```

The result should look like this:

![2015-04-01_17h56_56](https://cloud.githubusercontent.com/assets/873785/6953627/8a675e94-d898-11e4-8f80-f85d0a57857d.png)